benchmarks/2019.json

```diff
@@ -5183,6 +5273,24 @@
       100
     ]
   },
+  {
+    "measureId": "ACRAD5",
```

Comment:
Previous
```
{"level":"debug","message":"measure ACRAD5 2017 submission method registry single_performance_rate_measurements: 89 measurement-stratas","timestamp":"2019-01-03T23:37:46.754Z"}
{"level":"debug","message":"A total of 81 measurements may be in decile calculation for measure ACRAD5 registry 4","timestamp":"2019-01-03T23:37:46.762Z"}
{"level":"debug","message":"Success! measure ACRAD5 has 81 qualified TIN-NPIs reporting for submission method registry","timestamp":"2019-01-03T23:37:46.762Z"}
```
Need to see why it was removed

Reply:
Removed here
https://github.com/CMSgov/qpp-measures-data/pull/269/files#diff-19cc2b2b0a838f96141210f2b77688c9L4797